srv/salt/ceph/stage/iscsi/default.sls

```diff
@@ -1,4 +1,4 @@
 
 include:
-  - .core
-  - ...restart.igw.lax
```

Comment:
I might have overlooked it but I can't see any restart logic for igw.

Reply:
I removed it because I thought we would not need it. But now that I'm thinking again about this, we may need it for the case where we have updated the ceph-iscsi packages and need to restart the services. I'll add it back